config-linux.md

```diff
@@ -118,6 +118,21 @@ For more information, see [the man page](http://man7.org/linux/man-pages/man8/sy
 
 rlimits allow setting resource limits. The type is from the values defined in [the man page](http://man7.org/linux/man-pages/man2/setrlimit.2.html). The kernel enforces the soft limit for a resource while the hard limit acts as a ceiling for that value that could be set by an unprivileged process.
 
+## Linux user namespace mappings
+
+```
+    "userMappings": [
```

Comment:
Just matching what we have in spec_linux.go. We could even go with user_mappings.